guides/NEW_APP.md

```diff
@@ -0,0 +1,50 @@
+Creating a new app
+==================
+
+The following steps will guide you through add adding a new app to our system that is built by the CI system and deployed to the live environment.
+
+Prerequisites
+-------------
+
+In order to follow this guide you will need:
```

Comment:
Really really pedantic and can be ignored

```suggestion
To follow this guide you will need:
```
- Keeping the wording short and to the point, wherever possible makes it quicker and easier to read.